deploy/template.yaml

```diff
@@ -1193,6 +1199,177 @@ Resources:
       Threshold: 1
       ComparisonOperator: GreaterThanOrEqualToThreshold
 
+  ### FallbackEmail Definition
+  FallbackEmailFunction:
+    Type: AWS::Serverless::Function
+    DependsOn: FallbackEmailFunctionLogGroup
+    Properties:
+      FunctionName: !Sub "Ipvreturn-FallbackEmail-${AWS::StackName}"
+      Handler: FallbackEmailHandler.lambdaHandler
+      CodeUri: ../src/
+      ReservedConcurrentExecutions: !If
+        - ApplyReservedConcurrency
+        - !Ref LambdaConcurrency
+        - !Ref AWS::NoValue
+      Environment:
+        Variables:
+          POWERTOOLS_SERVICE_NAME: FallbackEmailHandler
+          SESSION_EVENTS_TABLE:
+            Fn::ImportValue: !Sub "${L2DynamoStackName}-session-events-table-name"
+          GOV_NOTIFY_QUEUE_URL: !Ref GovNotifySQSQueue
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSXrayWriteOnlyAccess
+        - DynamoDBWritePolicy:
+            TableName:
+              Fn::ImportValue:
+                !Sub "${L2DynamoStackName}-session-events-table-name"
+        - DynamoDBReadPolicy:
+            TableName:
+              Fn::ImportValue:
+                !Sub "${L2DynamoStackName}-session-events-table-name"
+        - KMSDecryptPolicy:
+            KeyId:
+              Fn::ImportValue:
+                !Sub "${L2DynamoStackName}-session-events-table-key-id"
+        - Statement:
+            - Effect: Allow
+              Action:
+                - "sqs:SendMessage"
+              Resource: !GetAtt GovNotifySQSQueue.Arn
+            - Effect: Allow
+              Action:
+                - "kms:Encrypt"
+                - "kms:Decrypt"
+                - "kms:GenerateDataKey"
+              Resource: !GetAtt GovNotifyEncryptionKey.Arn
+      Events:
+        ScheduledEvent:
+          Type: Schedule
+          Properties:
+            Schedule: cron(0 16 ? * MON-FRI *)  # This will run at 4pm UTC on weekdays (Monday to Friday) - To be changed back to 10am soon
```

Comment:
Running this today should be okay to send out emails to those who have missed them...But we should have a ticket created to update the DB query to only extract those records which are 2-3days older so that we dont sent emails out to those sessions which are yet to received the VC_CONCUMED event and we dont falsely send them this email? cc: @Sanka-dS

Reply:
Agreed @hemavana, I think today let's let it run then immediately after push another commit to prod to remove this Cron so it doesn't run tomorrow. In parallel let's raise that ticket and bring into next Amigos session - Does that sound good?

Reply:
Yes. That sound good. Thanks!